policies/linux.consul.-.catalog.total_nodes.json

```diff
@@ -0,0 +1,27 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "consul.catalog.total_nodes",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a WARNING event when the number of total nodes drops.",
+    "duration": 900,
+    "enabled": false,
+    "name": "Consul - Total Nodes Deviation",
```

Comment:
This title should indicate the direction of the deviation, if there is one. Even better might be something like "Consul - Unusually Low Number of Nodes".

Reply:
@Grumblesmith I've commited the requested changes to the branch